cli/registry/client/endpoint.go

```diff
@@ -30,32 +31,35 @@ func (r repositoryEndpoint) BaseURL() string {
 }
 
 func newDefaultRepositoryEndpoint(ref reference.Named, insecure bool) (repositoryEndpoint, error) {
+	repoName := reference.TrimNamed(ref)
 	repoInfo, _ := registry.ParseRepositoryInfo(ref)
-	endpoint, err := getDefaultEndpointFromRepoInfo(repoInfo)
+	indexInfo := repoInfo.Index
```

Comment:
In case you're wondering why the extra variables; some of these are as a placeholder / intermediate state while I'm cleaning up things elsewhere.